src/components/molecules/card/card.html

```diff
@@ -0,0 +1,11 @@
+<div class="card">
+  <p class="card-title">card-title</p>
+  <div class="vector"></div>
+  <figure class="card-image-wrap">
+    <img src="../../../img/card-1.jpg" class="card-image"> </img>
```

Comment:
```suggestion
    <img src="../../../img/card-1.jpg" class="card-image" />
```
imgは終了タグが必要ないので、このように書きましょう。